.github/workflows/all_solutions.yml

```diff
@@ -279,6 +280,7 @@ jobs:
       # Make this variable true to enable extra data-gathering and logging to help troubleshoot test failures, at the cost of additional time and resources
       enhanced_logging: false
       NR_DOTNET_TEST_SAVE_WORKING_DIRECTORY: 1
+      azure_func_exe_path: C:\ProgramData\chocolatey\lib\azure-functions-core-tools\tools\func.exe
```

Comment:
Is there a variable we can leverage so we don't have to list the full path? I could see it potentially changing with e.g. a new version of Chocolatey

Reply:
I'd much rather just rely on the azure tools being in the path but that apparently doesn't happen with chocolatey installs. I don't see any obvious way to determine the base path for chocolatey, so I think we'll just have to live with this for now and deal with any breakage that happens in the future.

Reply:
Actually it looks like there's a `ChocolateyInstall` environment variable I can leverage. Change coming soon..